homeassistant/components/ruckus_unleashed/device_tracker.py

```diff
@@ -95,17 +103,25 @@ def mac_address(self) -> str:
     @property
     def name(self) -> str:
         """Return the name."""
-        if self.is_connected:
-            return (
-                self.coordinator.data[API_CLIENTS][self._mac][API_NAME]
-                or f"{MANUFACTURER} {self._mac}"
-            )
-        return self._name
+        return (
+            self._name
+            if not self.is_connected
```

Comment:
We try to avoid ternary operator expressions that span more than one line. They are not very readable.